CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @nginxinc/kic
-/site/ @nginxinc/kic @nginxinc/nginx-docs
+* @nginx/kic
+/site/ @nginx/kic @nginx/nginx-docs
```

Comment:
github says the codeowners contains errors, I think we should fix this first

![image](https://github.com/user-attachments/assets/624a0a1d-b7f8-4dfb-a8e7-6ebd5ffde429)

Reply:
during the migration the groups also change their orgs?

Reply:
The groups can't be migrated, but I created a new group in the `nginx` org with the same name `kic`